deployment/stack.py

```diff
@@ -44,12 +46,22 @@ class ConfigError(Exception):
     pass
 
 
-def main():
+def main(argv=None):
+    if argv is None:
+        argv = sys.argv
+
     config, config_yaml = load_config()
 
     derive_config(config)
 
-    unique_suffix = datetime.datetime.utcnow().strftime("%Y%m%dT%H%M")
+    stack_name = config['STACK_NAME']
+
+    # :todo: expand this into a proper command parser.
+    # For now, just the "name" command,
+    # which we plan to remove, when the Makefile goes.
+    if len(argv) > 1 and argv[1] == "name":
+        print(stack_name)
```

Comment:
There is now :) https://github.com/refinery-platform/refinery-platform/wiki/Coding-Style-Guide